File: wordslines.py
#!/usr/bin/python
import sys
import re

nonletters = re.compile("[^a-zA-Z ]")

for line in sys.stdin:
  line = nonletters.sub( "", line )
  words = line.split()
  if len(words) > 0:
        print "LongValueSum:words\t{0:d}".format( len(words) )
        print "LongValueSum:lines\t1"

File: wordslines.pl
#!/usr/bin/perl

#"Usage: cat shakespeare.txt | wordslines.pl \n";

while (<STDIN>){
  chop;
  $_ =~ s/[^a-zA-Z ]//g;  # only letters and spaces remain after this.
  @words = split;
  $numwords = $#words + 1;
  if ($numwords > 0){
    print "LongValueSum:words\t$numwords\n";
    print "LongValueSum:lines\t1\n";
  }
}
